pkg/operator/controllers/workaround/systemreserved.go

```diff
@@ -87,6 +87,10 @@ func (sr *systemreserved) Ensure(ctx context.Context) error {
 			kc.Labels = make(map[string]string)
 		}
 		kc.Labels[labelName] = labelValue
+		if kc.Annotations == nil {
```

Comment:
This will blindly `Update` and won't check for existence of other kubeletconfigs that may have the same annotation. This will put MCO in an unknown condition.